content/influxdb/cloud/organizations/users.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Manage users
+seotitle: Manage users in your InfluxDB Cloud organization
+description: >
+  Learn how to invite, view, and manage users in your InfluxDB Cloud organization.
+weight: 106
+menu:
+  influxdb_cloud:
+    parent: Manage organizations
+    name: Manage Users
+aliases:
+  - /influxdb/v2.0/account-management/multi-user/
+  - /influxdb/cloud/account-management/multi-user/
+  - /influxdb/cloud/account-management/multi-user/invite-user/
+  - /influxdb/cloud/account-management/multi-user/remove-user/
+  - /influxdb/cloud/users/
+---
+
+{{< cloud-name >}} lets you invite and collaborate with multiple users in your organization. 
+By default, each user has full permissions on resources in your organization.
+
+- [Users management page](#users-management-page)
+- [Invite a user to your organization](#invite-a-user-to-your-organization)
+  - [Resend an invitation](#resend-an-invitation)
+  - [Withdraw an invitation](#withdraw-an-invitation)
+- [Remove a user from your organization](#remove-a-user-from-your-organization)
+  - [Remove yourself from an organization](#remove-yourself-from-an-organization)
+
+## Users management page
+Manage your organization's users from your organizations **Users management page**.
+In the {{< cloud-name "short" >}} user interface (UI), click your user avatar in the left
+navigation menu, and select **Users**.
+
+{{< nav-icon "account" >}}
+
+## Invite a user to your organization
+
+1.  Navigate to your organization's [Users management page](#users-management-page).
+2.  Under **Add a new user to your organization**, enter the email address of
+    the user to invite and select their role in your organization.
+
+    {{% note %}}
+#### Available roles
```

Comment:
Is the indentation in lines 42-47 intentional?

Reply:
Yeah, this nests the note block under the list item. Content inside of shortcodes doesn't know about it's surrounding context or where the shortcode falls within the indentation structure. So whenever you want to nest a shortcode inside of a line item, the shortcode itself should be nested under the line item, but the content inside the shortcode needs to be formatted without indentation. If it's indented, Hugo parses it as a code block (default markdown behavior).